static/js/containers/ProfileFormContainer.js

```diff
@@ -98,6 +98,12 @@ class ProfileFormContainer extends React.Component {
         dispatch(clearProfileEdit(username));
       });
     } else {
+      this.setState({}, () => {
```

Comment:
Can you add a comment here to explain what's going on and that we want to make sure `scrollIntoView` is executed after rerender?